cypress/integration/user-journey.js

```diff
@@ -0,0 +1,233 @@
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Search Ovid MEDLINE®')
+
+            cy.get('input[type=file]')
+                .selectFile('tests/test-abstract-ovid-test-sample-5.txt')
+
+            cy.get('#upload_button')
+                .contains('Upload', { matchCase: false })
+                .click()
+
+        // We should be on the Select exposure MeSH® terms page
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Select exposure MeSH® terms')
+
+        // Lets add some exposure terms in the textarea, click add, then move on to mediators
+
+            cy.get('#term_names').type('Public Health Systems Research;Humans');
+
+            cy.contains('Add', { matchCase: false })
+                .click()
+
+            cy.contains('Save and move on to select mediators', { matchCase: false })
+                .click()
+
+        // We should be on the Select mediator MeSH® terms page
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Select mediator MeSH® terms')
+
+        // Lets add terms in the textarea, click add, then move on to outcomes
+
+            cy.get('#term_names').type('Genetic Markers; Penetrance');
+
+            cy.contains('Add', { matchCase: false })
+                .click()
+
+            cy.contains('Save and move on to select outcomes', { matchCase: false })
+                .click()
+
+        // We should be on the Select outcome MeSH® terms page
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Select outcome MeSH® terms')
+
+        // Lets add outcome term in the textarea, click add, move on to genes
+
+            cy.get('#term_names').type('Neoplasm Metastasis;Eryptosis');
+
+            cy.contains('Add', { matchCase: false })
+                .click()
+
+            cy.contains('Save and move on to select Genes and Filters', { matchCase: false })
+                .click()
+
+        // We should be on the Select genes and filter page
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Select genes and filter')
+
+        // click the button to continue and get some results
+
+            cy.get('button.btn.btn-primary')
+                .contains('Search', { matchCase: false })
+                .click()
+
+        // We should be on the my list of Results page
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: My list')
+
+        // get the table and check it has the uploaded text file listed
+
+            cy.get('table')
+                .eq(0)
+                .find('tr')
+                .eq(1)
+                .should('not.be.empty')
+
+        // now we are gonna wait a minute...
+
+            cy.wait(60000)
```

Comment:
@Rodsy Do both tests take 1 minute for the results to appear?  Reduce the wait time for the simple test where possible.